### PR TITLE
CI: use ubuntu-20.04 if necessary

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
See https://github.com/ansible-collections/news-for-maintainers/issues/28 for details.

##### ISSUE TYPE
- Test Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

**Comment by felixfontein** [comment]
Similar changes in https://github.com/ansible-collections/community.dns/pull/127, https://github.com/ansible-collections/community.docker/pull/520, https://github.com/ansible-collections/community.hrobot/pull/67, https://github.com/ansible-collections/community.internal_test_tools/pull/82, https://github.com/ansible-collections/community.routeros/pull/139, and https://github.com/ansible-collections/community.sops/pull/127.

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -63,7 +63,15 @@ jobs:
...
-    runs-on: ubuntu-latest
+    # Ansible-test on various stable branches does not yet work well with cgroups v2.
+    # Since ubuntu-latest now uses Ubuntu 22.04, we need to fall back to the ubuntu-20.04
+    # image for these stable branches. The list of branches where this is necessary will
+    # shrink over time, check out https://github.com/ansible-collections/news-for-maintainers/issues/28
+    # for the latest list.
+    runs-on: >-
+      ${{ contains(fromJson(
+          '["stable-2.9", "stable-2.10", "stable-2.11", "stable-2.12", "stable-2.13", "stable-2.14", "milestone"]'
+      ), matrix.ansible) && 'ubuntu-20.04' || 'ubuntu-latest' }}
```
Comment (resmo) [inline .github/workflows/ansible-test.yml:74]: what about "devel"?

Comment (felixfontein) [reply .github/workflows/ansible-test.yml:74]: `devel` has https://github.com/ansible/ansible/pull/78550 itself in it, it does not need a backport of that PR :)

**Review by kristianheljas — APPROVED** [review]
Looks good!

**Review by Andersson007 — APPROVED** [review]

**Comment by felixfontein** [comment]
@resmo @kristianheljas @Andersson007 thanks for reviewing this!

**Comment by Andersson007** [comment]
@felixfontein thanks for the fix!

**Pull request merged**